src/controller/org.controller/index.js

```diff
@@ -488,7 +488,7 @@ router.post('/org/:shortname/user',
     }
   }
   #swagger.responses[200] = {
-    description: 'Returns the created user entry (with the secret)',
+    description: 'Returns the user created (with the secret)',
```

Comment:
This sounds odd to me, maybe swap user and created around so it's "Returns the created user"